src/io/setrlimit/imandrakit_io_setrlimit.ml

```diff
@@ -0,0 +1,33 @@
+type resource =
+  | RLIMIT_CORE  (** maximum size of a core file, in bytes *)
+  | RLIMIT_CPU  (** maximum amount of CPU time, in seconds, used by a process *)
+  | RLIMIT_DATA  (** maximum size of a data segment of the process, in bytes *)
+  | RLIMIT_FSIZE
+      (** maximum size of a file, in bytes, that may be created by a process *)
+  | RLIMIT_NOFILE
+      (** a number one greater than the maximum value that the system may assign to a newly-created descriptor *)
+  | RLIMIT_STACK
+      (** maximum  size of the initial thread's stack, in bytes. This might clash with OCaml's stack management. *)
+  | RLIMIT_AS
+      (**  maximum size of total available memory of the process, in bytes *)
+
+module Raw = struct
+  let resource_to_int = function
```

Comment:
It could be helpful to sync those numbers with the kernel's, e.g. these https://github.com/torvalds/linux/blob/6e4436539ae182dc86d57d13849862bcafaa4709/include/uapi/asm-generic/resource.h (see also their note on compatibility)

Reply:
ah!! the idea is exactly that this is not the actual value, it's indexed on the C side in an array :)